demo/runners/support/agent.py

```diff
@@ -672,11 +673,9 @@ async def register_did(
         did: str = None,
         verkey: str = None,
         role: str = "TRUST_ANCHOR",
-        cred_type: str = CRED_FORMAT_INDY,
+        cred_type: str = CRED_FORMAT_INDY or CRED_FORMAT_VC_DI,
```

Comment:
I think @dhh1128 is correct this is a bug

Reply:
well it will just be CRED_FORMAT_INDY, unless that is falsy, in which case it will be CRED_FORMAT_VC_DI
but that is also not what we want

```>>> False or 'hi'
'hi'
```
but yeah its still a bug

Reply:
@tra371 @supersonicwisd1 lets fix this today and see what we want

Reply:
its not a major bug - its just a default - its just confusing.  we probably should leave the original default